src/plugins/intel_gna/src/gna_device.cpp

```diff
@@ -389,6 +389,7 @@ const std::map<Gna2ErrorType, const std::string> GNADeviceHelper::errorReasons =
 const std::map<Gna2OperationType, const std::string> GNADeviceHelper::operationTypes = {
     {Gna2OperationTypeNone, "None"},
     {Gna2OperationTypeConvolution, "Convolution"},
+    {Gna2OperationTypeConvolutionDWSC, "Dwsc"},
```

Comment:
Is it proper name?
I saw in few places:
"Dwsc"
"DWSC"
"dwsc"

why not 
`ConvolutionDWSC`

Reply:
DWSC = depth-wise separable convolution, so I think it's enough

Reply:
I just wonder that we want the same convetion as for other operation types
which uses name after "Gna2OperationType"

Reply:
Notice this isn't a convention, it's not kept in every case.